app/views/play/level/ControlBarView.coffee

```diff
@@ -182,6 +189,51 @@ module.exports = class ControlBarView extends CocoView
     return if @session.get('state')?.difficulty is @lastDifficulty
     @render()
 
+  onScoresUpdated: (e) ->
+    #return unless @level.get('assessment') in ['open-ended']
+    mainScore = e.scores?[0]
```

Comment:
If this is an expectation, this should be mentioned in the schema, and/or put in a separate property in the event for clarity.